### PR TITLE
Correct typo of a word and a bracket.

### PR DESCRIPTION
**Pull request by kangtaku**

### CODE REVIEW

File: spring-cloud-task-docs/src/main/asciidoc/features.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ Spring Boot application configured to be a task (annotated with the `@EnableTask
 annotation).
 
 At the beginning of a task (before any `CommandLineRunner` or `ApplicationRunner`
-implementations have been executed, an entry in the `TaskRepository` is created recording
+implementations) have been executed, an entry in the `TaskRepository` is created recording
```
Comment (cppwfs) [inline spring-cloud-task-docs/src/main/asciidoc/features.adoc]: Nice catch but probably will remove the opening parenthesis instead.  I'll fix this on merge.

**Comment by cppwfs** [comment]
Noticed you removed the EOL at the end of the Features.adoc.  that is a requirement for our docs.  adding it back on merge.

**Comment by cppwfs** [comment]
LGTM.  Will make corrections enumerated above.  Thanks for the contribution!

**Comment by cppwfs** [comment]
Fixed comments above on merge , squashed, merged

**Comment by kangtaku** [comment]
Okay! See you next commit!

Thanks for good stuff~!

**Pull request closed without merging**